deps/debian.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+echo "installing python pip and nodejs npm"
+sudo apt-get -y install python-pip npm jq
+echo "installing azure cli 1.0"
+sudo npm install --no-optional -g azure-cli
+echo "installing azure cli 2.0"
+sudo pip install azure-cli
+echo "installing aws cli"
+sudo pip install awscli
+echo "installing nash"
+go get github.com/NeowayLabs/nash
```

Comment:
this line doesn't install the nash binary, right? To install nash in $GOPATH I think it should be:
```go get github.com/NeowayLabs/nash/cmd/...```
I would add the -u flag also to force use the latest nash version because klb is driving nash development, several features depend on master.